controller/model/identity_handlers.go

```diff
@@ -241,14 +248,34 @@ func (handler *IdentityHandler) InitializeDefaultAdmin(username, password, name
 	identity, err := handler.ReadDefaultAdmin()
 
 	if err != nil && !boltz.IsErrNotFoundErr(err) {
-		pfxlog.Logger().Panic(err)
+		return err
 	}
 
 	if identity != nil {
 		return errors.New("already initialized: Ziti Edge default admin already defined")
 	}
 
-	identityType, err := handler.env.GetHandlers().IdentityType.ReadByName(IdentityTypeUser)
+	if len(username) < minDefaultAdminUsernameLength {
```

Comment:
are all these settings configurable? i'm just happy my "dev" password is at least 4 chars long. i almost feel like security software could take the "you really should know better" attitude ? :)

Reply:
It's not currently configurable. I was just moving some code around here. I don't know if Andrew has plans to make it configurable at some point.